release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1089,6 +1089,13 @@ With this release, specifying an unhealthy pod eviction policy for pod disruptio
 
 For more information, see xref:../nodes/pods/nodes-pods-configuring.adoc#pod-disruption-eviction-policy_nodes-pods-configuring[Specifying the eviction policy for unhealthy pods].
 
+[id="ocp-4-14-nodes-cgroupv2-default"]
+==== Linux Control Groups version 2 is now default
+
+Beginning with {product-title} 4.14, new installs use Control Groups version 2 by default, also known as cgroup v2, cgroup2, or cgroupsv2. This enhancement includes many bug fixes, performance improvements, and the ability to integrate with new features. cgroup v1 is still used in upgraded clusters that have initial installation dates prior to {product-title} 4.14. cgroup v1 can still be used by changing the `Node` object to `v1`.
```

Comment:
From looking at how things are discussed/marked-up in the `xref`'d section below (https://docs.openshift.com/container-platform/4.14/nodes/clusters/nodes-cluster-cgroups-2.html#nodes-clusters-cgroups-2_nodes-cluster-cgroups-2), it seems like this more correctly should be stated as:
```suggestion
Beginning with {product-title} 4.14, new installs use Control Groups version 2 by default, also known as cgroup v2, cgroup2, or cgroupsv2. This enhancement includes many bug fixes, performance improvements, and the ability to integrate with new features. cgroup v1 is still used in upgraded clusters that have initial installation dates prior to {product-title} 4.14. cgroup v1 can still be used by changing the `cgroupMode` field in the `node.config` object to `v1`.
```

Reply:
done